src/data/checks.js

```diff
@@ -150,6 +150,28 @@ function bindFindArtistOrAlias(boundFind) {
   };
 }
 
+function bindFindThing(boundFind) {
+  return unknownRef => {
+    // Find findable Things in the order defined in keylessReplacerOrder
+    for (const boundFindFunc of keylessReplacerOrder.map(key => boundFind[key])) {
+      const thing = boundFindFunc(unknownRef, {mode: 'quiet'});
+      if (thing) {
+        return boundFindFunc(unknownRef);
+      }
+    }
+    // If we still didn't match anything, check if an artist alias was misused
+    const artistAlias = boundFind.artistAlias(unknownRef, {mode: 'quiet'});
+    if (artistAlias) {
+      // No need to check if the original exists here. Aliases are automatically
+      // created from a field on the original, so the original certainly exists.
+      const original = artistAlias.aliasedArtist;
+      throw new Error(`Reference ${colors.red(unknownRef)} is to an alias, should be ${colors.green(original.name)}`);
```

Comment:
We don't really like making single-use variables that are literally just equivalent to a plain property access. Like, maybe "original" is more meaningful to you here, but everywhere *else* it's called `<Artist>.aliasedArtist`, so we prefer to use the latter syntax for consistency's sake.

We'd also split this string interpolation into a couple lines for legibility, like so:

```suggestion
      throw new Error(
        `Reference ${colors.red(unknownRef)} is to an alias, ` +
        `should be ${colors.green(artistAlias.aliasedArtist.name)}`);
```